CLAUDE.md

```diff
@@ -0,0 +1,80 @@
+# Flight Control - Claude Development Guide
+
+## Common Bash Commands
```

Comment:
make integration-test
make lint
make lint-docs and make spellcheck-docs (for doc updates)